packages/expo-clipboard/src/ExpoClipboard.ts

```diff
@@ -0,0 +1,3 @@
+// Temporary for SDK 40 until we make our own native implementation
+import Clipboard from 'react-native/Libraries/Components/Clipboard/Clipboard';
+export default Clipboard;
```

Comment:
i think we need to wrap this because react-native's Clipboard API doesn't expose `getStringAsync()` - it's just `getString()`

so we should do something similar to `ExpoClipboard.web.js` and just call through to the react-native Clipboard API here instead.

Reply:
@brentvatne good catch! 

we wrap `getString` AND `setString` now and expose them with `getStringAsync` and `setStringAsync`. I did this bc the web implementation of `setStringAsync` returns whether or not the content was successfully copied, so when we rewrite the native implementations we can add that feature as well (meaning that method will need to be async, so better to just name it that now rather than introduce a breaking change in the next SDK)

Reply:
i think in order to be consistent with the native implementation that we're wrapping, the return type of `setString` should be `void`, because we can't do anything else on top of this implementation:

https://github.com/facebook/react-native/blob/503a6f4463f5d2f7576b33158c18d8d8e99f3291/React/CoreModules/RCTClipboard.mm#L29-L33